pkg/util/ipvs/ipvs.go

```diff
@@ -78,7 +78,7 @@ func (svc *VirtualServer) String() string {
 	return net.JoinHostPort(svc.Address.String(), strconv.Itoa(int(svc.Port))) + "/" + svc.Protocol
 }
 
-// RealServer is an user-oriented definition of an IPVS real server in its entirety.
+// RealServer is an user-oriented definition of an IPVS Destination in its entirety.
```

Comment:
I discussed with @thockin before and there is a question about "real server" vs. "destination", which one is better?

And, we decided to follow the concept of IPVS.

Reply:
Makes sense, changed back.